server/pages/analytics.js

```diff
@@ -14,23 +12,26 @@ import analytics from '../lib/analytics';
 // Compute the number of milliseconds between each call to recordTiming
 const THROTTLE_MILLIS = 1000 / config( 'statsd_analytics_response_time_max_logs_per_second' );
 
-const logAnalyticsThrottled = throttle( function( sectionName, duration ) {
+const logAnalyticsThrottled = throttle( function( sectionName, duration, target ) {
 	analytics.statsd.recordTiming( sectionName, 'response-time', duration );
+	if ( target ) {
+		analytics.statsd.recordCounting( sectionName, `target-${ target }` );
```

Comment:
Little suggestion: if we ever want to record counts for anything else that `target` build, it would be more convenient if the event path was `target.evergreen` rather than `target-evergreen`. With dots, it's easier to filter the events for Grafana charts.

Reply:
Sure thing! It's a dot now :)